### ISSUE TITLE
PR has release notes for a different library

### ISSUE THREAD

**Issue by inglesp** [dependabot/dependabot-core#2482]
**Package manager/ecosystem**
Python/pip-tools

**Updated dependency**
pyarrow

**What you expected to see, versus what you actually saw**
I expected to see the release notes for pyarrow.  Instead, I saw the release notes for pandas!

**Images of the diff or a link to the PR, issue or logs**
https://github.com/ebmdatalab/openprescribing/pull/2786

![image](https://user-images.githubusercontent.com/28734/91287450-ab5fb980-e787-11ea-8d64-69924d9459ba.png)

**Comment by inglesp** [comment]
I haven't found any release notes for pyarrow, but here's a history of commits: https://github.com/apache/arrow/commits/master/python.

**Comment by jeffwidman** [comment]
👋 Hey there! Sorry for the glacially slow response... we've been a bit buried the past few years, and only just now starting to dig through some of our backlog.

This issue is pretty old, and we've pushed a lot of bugfixes since then... is this still a valid / reproducible issue?

Also, this is a package metadata issue... we definitely aren't perfect in our code paths for fetching metadata, but across the ecosystems we support what we've found is that quite frequently the problem is the underlying dependency getting bumped publishes wrong metadata.

Anyway, if you're still seeing this let us know, otherwise we should close this.
